pkg/kustomize/kustomize_cli_test.go

```diff
@@ -0,0 +1,37 @@
+package kustomize_test
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/kustomize"
+)
+
+func TestKustomizeCLI_FindKustomize(t *testing.T) {
+	cwd, _ := os.Getwd()
+	tmpDir := filepath.Dir(filepath.Join(cwd, "..", "test_data", "kustomize_dummy", "base"))
```

Comment:
I think you put your test_data directory into the wrong package. It should be in the same package as the this test file, then you don't have to do the cwd stuff and going up one directory (..)